dags/export_dag.py

```diff
@@ -110,6 +110,8 @@ def get_boolean_env_variable(env_variable_name, default=True):
 dags_folder = os.environ.get('DAGS_FOLDER', '/home/airflow/gcs/dags')
 export_max_workers = os.environ.get('EXPORT_MAX_WORKERS', '5')
 export_batch_size = os.environ.get('EXPORT_BATCH_SIZE', '10')
+export_daofork_traces = os.environ.get('EXPORT_DAOFORK_TRACES', '--daofork-traces')
```

Comment:
There is an issue in Airflow that causes empty env variables to not be propagated to cluster nodes. As a workaround you can make them boolean variable (True|False). And construct the parameters string `EXPORT_TRACES_EXTRA_PARAMS = '{} {}'.format('--daofork-traces' if export_daofork_traces else '', '--genesis-traces' if export_genesis_traces else '')`